extensions/amp-a4a/0.1/template-config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @typedef {{
+      requestUrl: string,
+    }} */
+export let NetworkRegistryEntryDef;
+
+/** @typedef {Object<string, !NetworkRegistryEntryDef>} */
+export let NetworkRegistryDef;
+
+/** @const {!NetworkRegistryDef} */
+export const NetworkRegistry = {
```

Comment:
I think the default config for ad networks can be a feature added later. As I'm yet not sure it will be helpful or not.

This `amp-ad-custom` extension is mainly for demo or trial purpose. Any real ad network should consider have its own extension `amp-ad-network-a`.